src/inmanta/execute/runtime.py

```diff
@@ -649,6 +650,13 @@ def __init__(self, queue: QueueScheduler):
         self.queue.add_to_all(self)
         self.done = False
 
+    def _requeue_with_additional_requires(self, key: Hashable, waitable: ResultVariable) -> None:
```

Comment:
This is named as if it's meant to be a private method but it isn't. Remove the leading underscore?

Reply:
changed